pkg/ddc/juicefs/dataset.go

```diff
@@ -72,7 +72,7 @@ func (j *JuiceFSEngine) UpdateDatasetStatus(phase datav1alpha1.DatasetPhase) (er
 		if !reflect.DeepEqual(dataset.Status, datasetToUpdate.Status) {
 			err = j.Client.Status().Update(context.TODO(), datasetToUpdate)
 			if err != nil {
-				j.Log.Error(err, "Update dataset %s in namespace %s error", datasetToUpdate.Name, datasetToUpdate.Namespace)
+				j.Log.Error(err, "Update dataset error", datasetToUpdate.Name, datasetToUpdate.Namespace)
```

Comment:
I suggest no need to record Error in Line 75 . Line 86 has already record what the update function returns.
As  RetryOnConflict document says
_"If the update function returns a Conflict error, RetryOnConflict will try again. On a non-Conflict error, or if it retries too many times and gives up, RetryOnConflict will return an error to the caller."_
Besides, i think function utils.LoggingErrorExceptConflict(,,,) may be a good choice dealing with conflict error.

Reply:
Thank you for your careful review!